package.json

```diff
@@ -88,6 +83,36 @@
             ]
         },
         "menus": {
+            "commandPalette": [
```

Comment:
I don't think that you need this--just remove the commands from `contributes.commands`.

Reply:
The commands have to appear in contributes.commands if they are referenced elsewhere in package.json (which they are, in a different contributes.menus section)